.github/workflows/dart.yaml

```diff
@@ -47,8 +46,10 @@ jobs:
         if: ${{ matrix.sdk == 'stable' }}
 
       - run: dart pub get
+
       - run: dart pub get
         working-directory: pkgs/native_assets_cli/example/native_add/
```

Comment:
Is this necessary? I thought `pub get` recursively gets all the dependencies.

Reply:
`flutter pub get` does, `dart pub get` doesn't.

* https://github.com/dart-lang/pub/issues/3855